build.gradle

```diff
@@ -2,35 +2,36 @@
 buildscript {
 
     ext {
-        kotlin_version = "1.4.32"
+        kotlinVersion = '1.5.21'
         retrofitVersion = '2.9.0'
-        navigationVersion = "2.3.4"
+        navigationVersion = '2.3.4'
         koinVersion = '2.0.1'
-        parcelerVersion = "1.1.13"
-        roomVersion = '2.2.6'
-        firebaseAnalyticsVersion = '18.0.2'
-        firebaseCrashlyticsVersion = '17.4.1'
-        firebaseMessagingVersion = "20.2.0"
-        firebaseConfigVersion = "19.1.4"
-        adapterDelegatesVersion = "4.2.0"
-        glideVersion = "4.9.0"
-        navigationVersion = "2.3.4"
+        parcelerVersion = '1.1.13'
+        roomVersion = '2.3.0'
+        firebaseAnalyticsVersion = '19.0.0'
+        firebaseCrashlyticsVersion = '18.2.1'
+        firebaseMessagingVersion = '20.2.0'
+        firebaseConfigVersion = '19.1.4'
+        adapterDelegatesVersion = '4.2.0'
+        glideVersion = '4.9.0'
+        navigationVersion = '2.3.5'
     }
 
     repositories {
         google()
         jcenter()
-        maven { url "https://plugins.gradle.org/m2/" }
+        mavenCentral()
```

Comment:
This change is because of [the JCenter deprecation](https://stackoverflow.com/questions/66651640/jcenter-deprecation-impact-on-gradle-and-android) announcement. The Android plugin will warn that the `jcenter` repository is now read only and that no new packages will be uploaded to it, but I couldn't find equivalent upstreams for all packages on the Maven Central repo.

Reply:
We should remove the jcenter repository. The dependencies that are only available in jcenter are _com.yqritc:recyclerview-flexibledivider_ which isn't used in the app and _com.sylversky.fontreplacer:fontreplacer_ which we should remove(only used in the application class). FontReplacer should be removed because the library hasn't been updated in years and there is a better and official way of setting the font with [downloadable fonts](https://developer.android.com/guide/topics/ui/look-and-feel/downloadable-fonts).

Reply:
@lukstbit I've tried using the Downloadable Fonts interface from Android Studio, but it seems the [IBM Plex Sans font](https://fonts.google.com/specimen/IBM+Plex+Sans#standard-styles) doesn't show up in the searchable list :confused: Do you have any idea what might be the cause?

![image](https://user-images.githubusercontent.com/3010346/130277846-5718aa67-a29a-4016-b38f-c425293afdfd.png)

Reply:
@GabrielMajeri Yes, the font doesn't seem to be available online. Still, remove the library and it's usage in the application class. I'll look at the downloadable font to see if I can make it work, if not I'll make the edits to manually use the fonts(or most likely set it in the app theme/styles).

Reply:
Done. I've opened #70 so we don't forget about adding back the fonts.